CMakeLists.txt

```diff
@@ -2,4 +2,73 @@
 cmake_minimum_required(VERSION 2.8)
 project(csm)
 
-INSTALL(FILES package.xml DESTINATION share/csm)
+if(WIN32)
+  add_definitions(-DWINDOWS)
+  if(MSVC)
+    # SSE2 optimizations
+    add_definitions("/arch:SSE2")
+  endif()
+endif()
+
+find_package(Eigen3 REQUIRED)
```

Comment:
More of just curiosity, but I want to make sure if eigen is already available on every OS (that esp. ROS2 builds installers for). Is it safer to defined a dependency on it in `package.xml`?

Reply:
Ah, yeah.  I'll add it to package.xml

Reply:
Fixed